src/WordPressCodeExtractor.php

```diff
@@ -128,6 +128,13 @@ function ( $file, $key, $iterator ) use ( $exclude ) {
 							return false;
 						}
 
+						// Check for more complex paths, e.g. /some/sub/folder.
+						foreach( $exclude as $path_or_file ) {
+							if ( substr( $file->getPathname(), - strlen( $path_or_file ) ) === $path_or_file ) {
```

Comment:
There's probably a better way to do this, but I couldn't think of one right now.